sources/PseudoTerminal.m

```diff
@@ -326,11 +326,13 @@ + (NSInteger)styleMaskForWindowType:(iTermWindowType)windowType {
         case WINDOW_TYPE_BOTTOM_PARTIAL:
         case WINDOW_TYPE_LEFT_PARTIAL:
         case WINDOW_TYPE_RIGHT_PARTIAL:
-        case WINDOW_TYPE_NO_TITLE_BAR:
             return NSBorderlessWindowMask | NSResizableWindowMask;
 
         case WINDOW_TYPE_TRADITIONAL_FULL_SCREEN:
             return NSBorderlessWindowMask;
+            
+        case WINDOW_TYPE_NO_TITLE_BAR_ROUNDED:
+            return NSTitledWindowMask | NSFullSizeContentViewWindowMask | NSClosableWindowMask | NSMiniaturizableWindowMask | NSResizableWindowMask;
```

Comment:
Does it have to have closable and miniaturizable? Those buttons aren't visible so it's strange that it's set.